Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/Booking/BookingLocation.hs

```diff
@@ -16,32 +16,8 @@ module Domain.Types.Booking.BookingLocation where
 
 import Data.Aeson
 import Data.OpenApi (ToSchema)
-import Data.Time
+import qualified Domain.Types.Location as DLoc
 import EulerHS.Prelude hiding (id, state)
```

Comment:
why this file is still existing. BookingLocation has to be removed and we should have just Location thing